File: cpp/soma_vetores.cpp
#include <iostream>

using namespace std;

int main(){
    int n;

    cout << "Quantos valores vai ter cada vetor? ";
    cin >> n;

    int a[n], b[n], c[n];

    cout << "Digite os valores do vetor A:" << endl;

    for (int i=0; i<n; i++) {
        cin >> a[i];
    }

    cout << "Digite os valores do vetor B:" << endl;

    for (int i=0; i<n; i++) {
        cin >> b[i];
    }

    for (int i=0; i<n; i++) {
        c[i] = a[i] + b[i];
    }

    cout << "VETOR RESULTANTE:" << endl;

    for (int i=0; i<n; i++) {
        cout << c[i] << endl;
    }

    return 0;
}

File: python/soma_vetores.py
n: int

n = int(input("Quantos valores vai ter cada vetor? "))

a: [int] = [0 for x in range(n)]
b: [int] = [0 for x in range(n)]
c: [int] = [0 for x in range(n)]

print("Digite os valores do vetor A:")

for i in range(n):
	a[i] = int(input())

print("Digite os valores do vetor B:")

for i in range(n):
	b[i] = int(input())

for i in range(n):
	c[i] = a[i] + b[i]

print("VETOR RESULTANTE:")

for i in range(n):
	print(c[i])